internal/workflow_testsuite.go

```diff
@@ -511,6 +511,11 @@ func (t *TestWorkflowEnvironment) SignalWorkflow(name string, input interface{})
 	t.impl.signalWorkflow(name, input)
 }
 
+// SignalWorkflow sends signal to the currently running test workflow.
+func (t *TestWorkflowEnvironment) SignalChildWorkflow(workflowID, signalName string, input interface{}) error {
```

Comment:
Will it signal the parent? Or it doesn't have an ID?
If it does shouldn't it be called SignalByID?

Reply:
Yes, it could signal parent if you pass in parent workflow ID. So rename to SignalWorkflowByID sounds good.